services/federator/test/unit/Test/Federator/ExternalServer.hs

```diff
@@ -196,7 +198,7 @@ requestGalleySuccess =
           . runInputConst noClientCertSettings
           . runInputConst scaffoldingFederationDomainConfigs
           $ callInward Galley (RPC "get-conversations") aValidDomain (CertHeader cert) request
-      let expectedCall = Call Galley "/federation/get-conversations" "\"foo\"" aValidDomain
+      let expectedCall = Call Galley "/federation/get-conversations" [] "\"foo\"" aValidDomain
```

Comment:
I'm reviewing commit by commit, but would it make sense to have a test with a non-empty list for headers?

Reply:
Alright, I'll try to add a test with headers.